### PR TITLE
update Readme.md for clusterloader2 to replace APIResponsiveness with APIResponsivenessPrometheusSimple

### PR DESCRIPTION
**Pull request by sonyafenge**
ref https://github.com/kubernetes/perf-tests/pull/915

### CODE REVIEW

File: clusterloader2/README.md [hunk 1]
```diff
@@ -85,10 +85,11 @@ export CL2_ACCESS_TOKENS_QPS=5
 ## Measurement
 
 Currently available measurements are:
-- **APIResponsiveness** \
-This measurement creates summary for latency and number for server api calls.
+- **APIResponsivenessPrometheusSimple** \
+This measurement creates summary for latency and number for server api calls based on the data collected by the prometheus-operator.
```
Comment (wojtek-t) [inline clusterloader2/README.md:89]: s/prometheus-operator/prometheus server/

Comment (sonyafenge) [reply clusterloader2/README.md:89]: could you please clarify your comments? do you mean replace prometheus-operator to "prometheus-operator/prometheus server"?

Comment (wojtek-t) [reply clusterloader2/README.md:89]: yes - sorry for not being clear

Comment (sonyafenge) [reply clusterloader2/README.md]: thanks and just updated.

File: clusterloader2/README.md [hunk 2]
```diff
@@ -85,10 +85,11 @@ export CL2_ACCESS_TOKENS_QPS=5
 ## Measurement
 
 Currently available measurements are:
-- **APIResponsiveness** \
-This measurement creates summary for latency and number for server api calls.
+- **APIResponsivenessPrometheusSimple** \
+This measurement creates summary for latency and number for server api calls based on the data collected by the prometheus-operator/prometheus server.
```
Comment (wojtek-t) [inline clusterloader2/README.md:89]: Sorry - it seems I wasn't clear again.

I was simply suggesting to remove "prometheus-operator" and just leave prometheus server.
See lines 94-98 - those two should be consistent.

Comment (sonyafenge) [reply clusterloader2/README.md:89]: so what is the difference between APIResponsivenessPrometheusSimple and  APIResponsivenessPrometheus? if the description is same, why we need both of them?

Comment (wojtek-t) [reply clusterloader2/README.md:89]: The readme is misleading. 

The simple simple gets percentiles over the whole test run.
The non-simple - implements the window-based official definition: https://github.com/kubernetes/community/blob/master/sig-scalability/slos/api_call_latency.md

Comment (sonyafenge) [reply clusterloader2/README.md]: thanks for your information. just updated and added percentiles to APIResponsivenessPrometheusSimple

**Pull request merged**